examples/example_otap.py

```diff
@@ -7,6 +7,27 @@
 import wirepas_mesh_messaging as wmm
 import logging
 from random import choice
+import sys
+
+otap_delay_choices = [
```

Comment:
Initially I didn't really give it as an option because I'm not sure it is clear for our customer which value to choose.
But if you gave good answer to give when someone will ask if it must use 10 or 30 minutes, go for it :-)

Reply:
No clear guidelines to give on this point unfortunately but as this script starts to look more like a reference one rather than an example leaving the delay hardcoded feels a bit weird.